### ISSUE TITLE
2.20 wording

### ISSUE THREAD

**Issue by relaxnow** [OWASP/ASVS#88]
> ... to prevent automated attacks against common authentication attacks such as brute force attacks or denial of service attacks.

Lot of 'attacks' in that sentence.
Also I don't think an attacker attacks common authentication attacks.

Maybe reword this to:
> Verify that request throttling is in place to prevent automated attacks such as brute force or denial of service attacks.

**Comment by vanderaj** [comment]
Thank you. Fixed.
